lib/jwt/jwk/key_base.rb

```diff
@@ -11,6 +11,7 @@ def initialize(keypair, kid = nil)
       end
 
       def self.inherited(klass)
+        super
```

Comment:
Dunno what the guidelines say, but feels like super should be the last thing happening in this method.